frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -195,34 +228,39 @@ const Search: React.FC<{
 		>
 			<IconSolidSearch className={styles.searchIcon} />
 
-			<Combobox
-				autoSelect={autoSelect}
```

Comment:
Encountered infinite rerenders when using autoSelect with `formState.setValue` in `onChange`